pkg/proxy/ipvs/proxier.go

```diff
@@ -952,7 +952,6 @@ func (proxier *Proxier) syncProxyRules(reason syncReason) {
 			Scheduler: proxier.ipvsScheduler,
 		}
 		// Set session affinity flag and timeout for IPVS service
-		var flags utilipvs.ServiceFlags
```

Comment:
the var `flags` is used in somewhere below. Since you remove it, please remove them as well.